src/Controller/GraphController.php

```diff
@@ -19,19 +20,22 @@ class GraphController
     private Parser $requestParser;
     private bool $shouldHandleCORS;
     private bool $useApolloBatchingMethod;
+    private bool $debugMode;
 
     public function __construct(
         BatchParser $batchParser,
         Executor $requestExecutor,
         Parser $requestParser,
         bool $shouldHandleCORS,
-        string $graphQLBatchingMethod
+        string $graphQLBatchingMethod,
+        bool $debugMode
```

Comment:
@mcg-web The current behaviour is to throw the exception always, so for BC debugmode should be `true`. If OK I'll change that asap. Thanks!

Reply:
maybe I missed some thing but throwing `BadResquestException` in production environment (debug mode false), any message will not be shown normally ? Only the http status will be public. This is the behavior we waiting for here.

Reply:
@mcg-web If you want it to default to always throw bad request exception then debugMode should default to `true`. (old behavior, not a BC break)

If you want it to default to NOT throw bad request exception, only the http status will be public, then debugMode should default to `false`. (new behavior, BC break)

Reply:
@mcg-web To be clear: The purpose of the PR is to prevent query parse errors from leading to visible stack traces on production. If the debugmode parameter is missing and it falls back to a default value of `false`, then query parse errors will never be thrown.

The old/current code does produce visible stack traces on production. If the debugmode parameter is missing and it falls back to a default value of `true`, then we get this backwards compatible behaviour: the exception is always thrown on query parse errors. That's why I suggested the value should be true.

But I'll change it to whatever you prefer. Please let me know. Thanks!

Reply:
@mcg-web I'll commit your suggestion as is, then. Cheers!